src/power_grid_model_io/mappings/tabular_mapping.py

```diff
@@ -19,8 +19,11 @@ class TabularMapping:
     Tabular data mapping helper class
     """
 
-    def __init__(self, mapping: Tables):
-        self._log = structlog.get_logger(type(self).__name__)
+    def __init__(self, mapping: Tables, logger=None):
+        if logger is None:
+            self._log = structlog.get_logger(type(self).__name__)
```

Comment:
no `id` here?

Reply:
Hahaha, it can have one but I chose not to because these few mapping classes are only referenced in the tabularconverter, which has pre-set way of usage. Can be ofc copy-pasted to give them an unique id.

Reply:
i guess it is ok like this